Makefile

```diff
@@ -20,9 +21,9 @@ clean:
 	rm -rf $(BUILDDIR)/*
 
 html:
-	$(SPHINXBUILD) -b html $(SPHINXOPTS) $(BUILDDIR)
+	$(SPHINXBUILD) -b html $(SPHINXOPTS) "$(SOURCEDIR)" "$(BUILDDIR)"
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)."
 
 livehtml:
-	sphinx-autobuild -b html --poll $(SPHINXOPTS) $(BUILDDIR)
+	sphinx-autobuild -b html --poll $(SPHINXOPTS) $(SOURCEDIR) $(BUILDDIR) -B -p 8001
```

Comment:
The port change broke docs serving from within Vagrant (I tend to run there to avoid installing the deps on the host); I've opened port 8001 in #3835. (And also updated the docs build directory path in `.gitignore`)

Reply:
Rather embarassingly the port part was an accidental check in!  Thanks for fixing :)